app/src/androidTest/java/com/github/polybooks/MainTest.kt

```diff
@@ -68,11 +68,19 @@ class MainTest {
 
     }
 
+//    @Test
+//    fun databaseButton() {
+//
+//        onView(withId(R.id.button_open_db_tests)).perform(click())
+//        Intents.intended(IntentMatchers.hasComponent(ListSalesActivity::class.java.name))
+//
+//    }
+
```

Comment:
I think that's a case where it's legitimate to not keep it as a comment. The purpose of the button has been changed, and it's anyway basic code with no logic.

Reply:
done